lib/semantics/check-block.cc

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2019, ARM Ltd.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <iostream>
+
+#include "../parser/message.h"
+#include "../parser/parse-tree-visitor.h"
```

Comment:
FYI, the include order does not follow the style guide, we use clang-format 7.1 to do that and other style stuff for us before PRs.

Reply:
Ok